app/externalapi.py

```diff
@@ -0,0 +1,50 @@
+
+import requests
+
+def headlines_feed():
```

Comment:
I would recommend to encapsulate all these functions in a class and use that class elsewhere in the application. This kind of philosophy can greatly improve the structure of your project. Also it forces you to give a more meaningful name to your file (which in this case "externalapi" doesn't necessarily tell me what api this is and what is it about the external api in this file - you definitely don't hold the api in the file :) )